examples/README.md

```diff
@@ -31,8 +31,18 @@ ize tunnel up
 ize tunnel down
 ```
 
+### Connect to a container in the ECS
+```shell
+ize console goblin
+```
+
 ### Upload/Remove secrets
 ```shell
-ize secret set --file .infra/env/testnut/secrets/example-service.json --type ssm
-ize secret remove --type ssm --path /testnut/example-service
+ize secrets set --file .infra/env/testnut/secrets/example-service.json --type ssm
+ize secrets rm --backend ssm --path /testnut/example-service
```

Comment:
```
This command removes secrets from storage

Usage:
  ize secrets rm <app> [flags]

Examples:
  # Remove secrets:
  
  # This will remove your secrets for "squibby" app
  ize secrets rm squibby

```